### PR TITLE
feat(exp): backup & restore e2e-test for zfs-localpv

### LINKED ISSUE #517: Automate E2e test for zfs-localpv backup & restore

### PR DESCRIPTION
**Pull request by w3aman**
Signed-off-by: Aman Gupta <aman.gupta@mayadata.io>

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #517 

**What this PR does / why we need it**:

### CODE REVIEW

File: experiments/zfs-localpv/functional/backup_and_restore/backup_restore.yml [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+      args:
+        executable: /bin/bash
+
+    when: lookup('env','RESTORE_IN_DIFF_NAMESPACE') == "true"
+
+  - block:
+
+    - name: Delete all the resources in application namespace
+      shell: | 
+        kubectl delete deploy,pvc,svc --all -n {{ app_ns }}
```
Comment (gprasath) [inline experiments/zfs-localpv/functional/backup_and_restore/backup_restore.yml:43]: should we consider backing up sts also

**Review by nsathyaseelan — APPROVED** [review]
lgtm

File: experiments/zfs-localpv/functional/backup_and_restore/run_litmus_test.yml [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+          - name: VELERO_PLUGIN_NAME  ## Velero plugin name. For zfs-localpv plugin name is `openebs/velero-plugin:2.1.0`
+            value: ''                 ## you can use ci image also. 
+
+          - name: VELERO_SCHEDULE_BACKUP_NAME  ## If you want to create scheduled backup (based on ACTION env) give the name for it.
+            value: ''
+
+          - name: VELERO_VERSION   ## Velero version (for e.g. v1.4.0)
+            value: ''
+
+          - name: STORAGE_BUCKET  ## Supported values: minio
```
Comment (gprasath) [inline experiments/zfs-localpv/functional/backup_and_restore/run_litmus_test.yml:60]: is minio alone supported?

Comment (w3aman) [reply experiments/zfs-localpv/functional/backup_and_restore/run_litmus_test.yml:60]: As of this automated experiment only minio...will add other buckets also after verifying

**Pull request merged**